sdk/sphere/Azure.ResourceManager.Sphere/api/Azure.ResourceManager.Sphere.netstandard2.0.cs

```diff
@@ -0,0 +1,611 @@
+    }
+    public partial class CertificateProperties
+    {
+        internal CertificateProperties() { }
+        public string Certificate { get { throw null; } }
+        public System.DateTimeOffset? ExpiryUtc { get { throw null; } }
+        public System.DateTimeOffset? NotBeforeUtc { get { throw null; } }
+        public Azure.ResourceManager.Sphere.Models.ProvisioningState? ProvisioningState { get { throw null; } }
+        public Azure.ResourceManager.Sphere.Models.CertificateStatus? Status { get { throw null; } }
+        public string Subject { get { throw null; } }
+        public string Thumbprint { get { throw null; } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct CertificateStatus : System.IEquatable<Azure.ResourceManager.Sphere.Models.CertificateStatus>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public CertificateStatus(string value) { throw null; }
+        public static Azure.ResourceManager.Sphere.Models.CertificateStatus Active { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.CertificateStatus Expired { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.CertificateStatus Inactive { get { throw null; } }
+        public static Azure.ResourceManager.Sphere.Models.CertificateStatus Revoked { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Sphere.Models.CertificateStatus other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Sphere.Models.CertificateStatus left, Azure.ResourceManager.Sphere.Models.CertificateStatus right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Sphere.Models.CertificateStatus (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Sphere.Models.CertificateStatus left, Azure.ResourceManager.Sphere.Models.CertificateStatus right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class ClaimDevicesContent
+    {
+        public ClaimDevicesContent(System.Collections.Generic.IEnumerable<string> deviceIdentifiers) { }
+        public System.Collections.Generic.IList<string> DeviceIdentifiers { get { throw null; } }
+    }
+    public partial class CountDeviceResponse : Azure.ResourceManager.Sphere.Models.CountElementsResponse
+    {
+        internal CountDeviceResponse() { }
+    }
+    public partial class CountElementsResponse
```

Comment:
Shall we change the suffix Response to Result?

Reply:
fixed in latest commit